python/dgl/_ffi/_ctypes/object.py

```diff
@@ -40,9 +40,10 @@ def _return_object(x):
 class ObjectBase(object):
     """Object base class"""
     __slots__ = ["handle"]
+
     # pylint: disable=no-member
     def __del__(self):
-        if _LIB is not None:
+        if _LIB is not None and hasattr(self, 'handle'):
```

Comment:
When would this doesn't have `handle`?

Reply:
Possible in a multi-processing setting. When a worker process creates an object by `__new__` but before calling `__init_handle_by_constructor` to create `handle`, it exits due to program exit, which will call `__del__` with no `handle` created.